src/backend/assets/js/main.js

```diff
@@ -112,8 +112,46 @@ const loadVideoAndSlides = function(div, videoURL, slideURL) {
   }
 };
 
-window.handleClientLoad = handleClientLoad;
-window.initClient = initClient;
-window.handleAuthClick = handleAuthClick;
-window.listUpcomingEvents = listUpcomingEvents;
-window.loadVideoAndSlides = loadVideoAndSlides;
+$(document).ready(function () {
+  let filter;
+  //store room filters
+  let room = {
+    dateFilter:'.date-filter'
+  };
+  //store track filters
+  let track = {
+    dateFilter:'.date-filter'
+  };
+  //store schedule filters
+  let schedule = {
+    dateFilter:'.day-filter'
+  };
```

Comment:
OK I see what you are trying to do here. The use of `eval` is strongly unrecommended. Here is what you can do instead:

```js
const allFilters = {
  room: {
    dateFilter: '.date-filter'
  },
  track: {
    dateFilter: '.date-filter'
  },
  schedule: {
    dateFilter: '.date-filter'
  }
};
```

Now we have a well-defined object with the filters in it. And so...